hips/hip-0012.md

```diff
@@ -1,38 +1,225 @@
 ---
 hip: "0012"
 title: "Helm 4 Development Process"
-authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>", "George Jenkins <gvjenkins@gmail.com>", "Matt Farina" ]
 created: "2021-04-02"
 type: "process"
 status: "draft"
 ---
 
 ## Abstract
 
-The Helm 4 development lifecycle will be managed by a release engineer.
-It will be kicked off in October, 2021.
-The process will last not less than six months.
-There will be an active development phase, followed by an alpha period, a beta period, and a release candidate period.
-The release manager will determine when Helm 4 is ready to ship.
+
+Helm 4 is planned as the next major version release of Helm.
+With KubeCon/CloudNativeCon North America 2024 marking the five year anniversary since the successful Helm 3 release.
+The development of Helm for is strongly desired in order to continue to evolve Helm and its ecosystem.
+
+This HIP documents the requirements and process for building Helm 4.
+Describing the scope of Helm 4 changes.
+With particular concern to maintaining continuality in the Helm ecosystem.
+
+This HIP also aims for an open and productive process that enables Helm 4 to build on the success of Helm 3 and to continues to evolve Helm for the success of the Helm communitity.
+
+
+
+<!--
+
+Changes are to be managed by accompanying (lighter-weight) "Helm 4 proposal", or standard Helm HIP.
+
+To be approved by Helm maintainers.
+Once approved, enabling the related changes to be approved/merged by a release engineer.
+In order to ease the transtions for users, behavioural changes will need to be accompained by release notes and migration guides, etc.
+
+
+Preventing a protacted process that prevents the release of Helm 4 in a timely fashion (there will be a Helm 5 of course!)
+
+The release engineer will determine when Helm 4 is ready to ship. And once shipped, the defined deprecation phase for Helm 3 will begin.
+-->
 
 ## Motivation
 
-We do not currently have a documented process for kickstarting new major Helm versions.
-During the Helm 3 process, we drew criticism for not having such documentation.
-However, a "generalized" process that can span multiple major versions over multiple
-years is ambitious.
-So this proposal attempts to merely spell out a formal process for Helm 4.
+Helm 3 has been greatly sucessful.
+And Helm has built a significant ecosystem of the Helm tooling (CLI), charts, chart repositories and repos, the SDK, plugins, etc.
+The Helm project needs to continue to evolve.
+Kubernetes itself has continued to grow and change.
+And Helm needs to continue to evolve as well to continue to provide value to the Helm community.
+
+However, Helm has also accumulated debt.
+And Helm 3 today has the significant issue that many (good) changes are overly difficult to implement due to compatibility concerns: either breakage of behavioural functionality or API incompatibility.
```

Comment:
This might be a good place to mention SemVer and how breaking and behavioral changes cannot happen without a major version change.